microbench/edit_scalability.cpp

```diff
@@ -1,14 +1,11 @@
-#include "llvm/Support/CommandLine.h"
-namespace cl = llvm::cl;
-
 #include <fstream>
 #include <iostream>
 #include <mutex>
 #include <sstream>
 #include <utility>
 #include <vector>
-using namespace std;
 
+#include "llvm/Support/CommandLine.h"
```

Comment:
we will not support llvm as command line, please use boost.

Reply:
this should be done by the original author

Reply:
@meyerzinn please add this to your list of chores.

Reply:
then once you have seen this resolve this